fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -217,6 +202,33 @@ private fun PlaceholderBookmarkImage() {
     )
 }
 
+@Composable
+private fun FaviconFallbackBookmarkImage(
+    bookmark: RecentBookmark,
+) {
+    if (!bookmark.url.isNullOrEmpty()) {
+        components.core.icons.Loader(bookmark.url) {
```

Comment:
I am wondering if we could replace this with https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/compose/Favicon.kt

Reply:
This can also be a followup task.

Reply:
Didn't know we have this, thanks for letting me know! Added a box in order to respect the UX layout, and we can just use `Favicon`.